src/engine/Makefile

```diff
@@ -24,6 +24,7 @@ EE_OBJS =								\
 	modules/vif_sender.o				\
 	utils/math.o						\
 	utils/string.o						\
+	utils/Tyra_version.o				\
```

Comment:
Please do it `tyra_version`.

Pretty cool idea, overall!

Reply:
When it comes to tyrav2.
I think that the good place will be `/info/version.hpp` and we can include this class in `/info.hpp`

Reply:
> When it comes to tyrav2. I think that the good place will be `/info/version.hpp` and we can include this class in `/info.hpp`

Yes it sounds good